api-studdybuddies-backend.js

```diff
@@ -19,6 +20,7 @@ parametros.cargarParametros()
 
         // Añadimos las rutas
         app.use('/api/v1/user', userRoute);
+        app.use('/api/v1/room', roomRoute);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7172443)
```suggestion
        app.use("/api/v1/room", roomRoute);
```